3ds Max/Max2Babylon/Exporter/BabylonExporter.Material.cs

```diff
@@ -403,14 +403,20 @@ private void ExportPhysicalMaterial(IIGameMaterial materialNode, IIPropertyConta
             var propertyContainer = materialNode.IPropertyContainer;
 
             bool isTransparencyModeFromBabylonAttributes = false;
+            bool usePbrFactor = false; // this force the exporter to set the metallic or roughness even if the map are set
             if (babylonAttributesContainer != null)
             {
-                IIGameProperty babylonTransparencyModeGameProperty = babylonAttributesContainer.QueryProperty("babylonTransparencyMode");
-                if (babylonTransparencyModeGameProperty != null)
+                IIGameProperty gameProperty = babylonAttributesContainer.QueryProperty("babylonTransparencyMode");
+                if (gameProperty != null)
                 {
-                    babylonMaterial.transparencyMode = babylonTransparencyModeGameProperty.GetIntValue();
+                    babylonMaterial.transparencyMode = gameProperty.GetIntValue();
                     isTransparencyModeFromBabylonAttributes = true;
                 }
+                gameProperty = babylonAttributesContainer.QueryProperty("babylonUseFactors");
```

Comment:
I'm a bit confused, why do we want this flag? if a map is being used for O/R/M, don't we want to export them instead of the factors?

Reply:
Max behavior is Map OR scalar, but gltf and Babylon use these factor to also multiply the values of the map. According to the GLTF 2.0 Spec "If both factors and textures are present the factor value acts as a linear multiplier for the corresponding texture values."

Reply:
Makes sense, thanks for the explanation.